modules/network/src/integrationTest/java/org/apache/ignite/internal/network/netty/ItConnectionManagerTest.java

```diff
@@ -308,7 +305,26 @@ public void testConnectMisconfiguredClient() throws Exception {
     public void testStartTwice() throws Exception {
         ConnectionManagerWrapper server = startManager(4000);
 
-        assertThrows(IgniteInternalException.class, server.connectionManager::start);
+        IgniteInternalException exception = (IgniteInternalException) assertThrows(
```

Comment:
How about making `assertThrows()` generic (as it is in `Assertions`) to get rid of this ugly cast?

Reply:
I like that there is a check for a fragment of the error text. The cast does not bother me yet.